vyxal/structure.py

```diff
@@ -9,9 +13,7 @@ def __init__(self, *branches: Branch):
         self.branches = branches
 
     def transpile(self) -> str:
```

Comment:
I don't think we need this anymore, honestly. Transpilation should be separated from the definitions of the structures (and it already is anyway).

Reply:
Yeah, I thought that too, but that's a PR for a different day I think